pkgs/by-name/st/stalwart-mail/package.nix

```diff
@@ -102,6 +102,12 @@ rustPlatform.buildRustPackage rec {
     "--skip=smtp::inbound::data::data"
     # Expected "X-My-Header: true" but got Received: from foobar.net (unknown [10.0.0.123])
     "--skip=smtp::inbound::scripts::sieve_scripts"
+    # thread 'smtp::outbound::lmtp::lmtp_delivery' panicked at tests/src/smtp/session.rs:313:13:
```

Comment:
Is there a test supposed to be disabled after this comment?

Reply:
No the next line is from the same error message (same as with the other two comments that belong to one failing test)